.env.example

```diff
@@ -1,4 +1,4 @@
 RPC_1=
-CONSENSUS_RPC_1=
-BEACONCHAIN_API_KEY_1=
-BEACONCHAIN_API_URL_1=https://beaconcha.in
\ No newline at end of file
+CONSENSUS_RPC_URL=
+BEACONCHAIN_API_KEY=
+BEACONCHAIN_API_URL=https://beaconcha.in
```

Comment:
Should we remove this since I just got rid of our subscription lol

Reply:
yeah we can